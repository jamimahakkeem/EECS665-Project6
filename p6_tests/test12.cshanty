void main() {
	int i;
	i = 0;
	while(i < 10) {
		i++;
	}
	bool b;
	b = true;
}