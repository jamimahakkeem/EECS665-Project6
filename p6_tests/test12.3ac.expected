[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
i (local var of 8 bytes)
b (local var of 8 bytes)
tmp0 (tmp var of 8 bytes)
[END main LOCALS]
main:       enter main
            [i] := 0
lbl_1:      nop
            [tmp0] := [i] LT64 10
            IFZ [tmp0] GOTO lbl_2
            [i] := [i] ADD64 1
            goto lbl_1
lbl_2:      nop
            [b] := 1
lbl_0:      leave main
